aiidalab_widgets_base/computers.py

```diff
@@ -693,14 +698,18 @@ def __init__(self, text='Select computer:', path_to_root='../', **kwargs):
         self._refresh()
         super().__init__(children=children, **kwargs)
 
-    @staticmethod
-    def _get_computers():
+    def _get_computers(self):
         """Get the list of available computers."""
-        query_b = QueryBuilder()
-        query_b.append(Computer, tag='computer')
+        querybuild = QueryBuilder()
```

Comment:
While I'm generally in favor of very explicit (and therefore usually long) variable names, this is a moment, where I'd prefer a short one to make the following code more readable.
```suggestion
        qb = QueryBuilder()
```

Reply:
agreed.